utils/sanitiseMetadata.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable camelcase*/
+const { omit, assign } = require("lodash/fp");
+const uuid = require("uuid/v1");
+
+const filterUnacceptableMeta = omit([
+  "survey_url",
+  "tx_id",
+  "iat",
+  "exp",
+  "eq_id",
+  "form_type",
+  "account_service_url"
+]);
+
+const defaultMetadata = (questionnaireId, tokenIssueTime, surveyUrl) => ({
+  tx_id: uuid(),
+  jti: uuid(),
+  iat: tokenIssueTime,
+  exp: tokenIssueTime + 10000,
```

Comment:
Can we extract this `10000` as a constant - I presume its the expiry time but in what unit. Is it 10000 seconds or milliseconds?